### PR TITLE
Only publish tags to npm

### PR DESCRIPTION
**Pull request by armanbilge**

### CODE REVIEW

**Review by toddburnside — APPROVED** [review]
Thanks! No more going to find out why the build failed just to find it was this. 🎉

File: build.sbt [hunk 1]
```diff
@@ -166,14 +166,15 @@ ThisBuild / githubWorkflowPublishPreamble +=
...
       "registry-url" -> "https://registry.npmjs.org"
     )
   )
 
 ThisBuild / githubWorkflowPublish ++= Seq(
   WorkflowStep.Sbt(List("npm/npmPublish"),
                    name = Some("NPM Publish"),
-                   env = Map("NODE_AUTH_TOKEN" -> s"$${{ secrets.NPM_REPO_TOKEN }}")
+                   env = Map("NODE_AUTH_TOKEN" -> s"$${{ secrets.NPM_REPO_TOKEN }}"),
+                   cond = Some("startsWith(github.ref, 'refs/tags/v'")
```
Comment (toddburnside) [inline build.sbt:178]: Is it missing a closing paren here?

Comment (toddburnside) [reply build.sbt:178]: Are you sure? It seems to be one short in ci.yml

Comment (armanbilge) [reply build.sbt:178]: Ohhh in the workflow! Yes there's a paren missing there, good eye. I was just trying to figure out why CI didn't run 😅

**Pull request merged**